tests/helpers/test_reports_helpers.py

```diff
@@ -175,21 +195,68 @@ def test_map_labware_to_location_dataframe_content(app, freezer, labwhere_sample
 
 
 def test_add_cherrypicked_column(app, freezer):
-    # mocks response from get_cherrypicked_samples()
+    # existing dataframe before 'add_cherrypicked_column' is run (essentially queried from MongoDB)
     existing_dataframe = pd.DataFrame(
-        [["MCM001", "123", "TEST"], ["MCM002", "123", "TEST"], ["MCM003", "123", "TEST"]],
-        columns=[FIELD_ROOT_SAMPLE_ID, FIELD_PLATE_BARCODE, "Lab ID"],
+        [
+            ["MCM001", "123", "TEST", "Positive", "A1"],
+            ["MCM001", "456", "TEST", "Positive", "A1"],  # plate barcode differs from first sample
+            ["MCM001", "123", "TEST", "Positive2", "A1"],  # result differs from first sample
+            ["MCM001", "123", "TEST", "Positive", "A2"],  # coordinate differs from first sample
+            ["MCM002", "123", "TEST", "Positive", "A1"],  # root sample id differs from first sample
+        ],
+        columns=[
+            FIELD_ROOT_SAMPLE_ID,
+            FIELD_PLATE_BARCODE,
+            "Lab ID",
+            FIELD_RESULT,
+            FIELD_COORDINATE,
+        ],
     )
 
+    # mock response from the 'get_cherrypicked_samples' method
+    mock_get_cherrypicked_samples_rows = [
+        ["MCM001", "123", "Positive", "A1"],  # matches first sample only
+        ["MCM002", "123", "Positive", "A1"],  #  matches final sample only
+    ]
+    mock_get_cherrypicked_samples_columns = [
+        FIELD_ROOT_SAMPLE_ID,
+        FIELD_PLATE_BARCODE,
+        FIELD_RESULT,
+        FIELD_COORDINATE,
+    ]
     mock_get_cherrypicked_samples = pd.DataFrame(
-        ["MCM001", "MCM003"], columns=[FIELD_ROOT_SAMPLE_ID]
+        np.array(mock_get_cherrypicked_samples_rows), columns=mock_get_cherrypicked_samples_columns
     )
 
-    expected_columns = [FIELD_ROOT_SAMPLE_ID, FIELD_PLATE_BARCODE, "Lab ID", "LIMS submission"]
+    # output from 'add_cherrypicked_column' - after merging existing_dataframe with response from 'get_cherrypicked_samples'
+    expected_columns = [
+        FIELD_ROOT_SAMPLE_ID,
+        FIELD_PLATE_BARCODE,
+        "Lab ID",
+        FIELD_RESULT,
+        FIELD_COORDINATE,
+        "LIMS submission",
+    ]
     expected_data = [
-        ["MCM001", "123", "TEST", "Yes"],
-        ["MCM002", "123", "TEST", "No"],
-        ["MCM003", "123", "TEST", "Yes"],
+        [
+            "MCM001",
+            "123",
+            "TEST",
+            "Positive",
+            "A1",
+            "Yes",
+        ],  # 'Yes' because was returned from get_cherrypicked_samples
+        ["MCM001", "456", "TEST", "Positive", "A1", "No"],
```

Comment:
Very minor thing, why are these 3 on one line but the first and final spaced out?